server/db/repositories/helpers.go

```diff
@@ -133,12 +136,22 @@ func compileQuery(collectionName string, q dal.Query) dal.CompiledQuery {
 		}
 	}
 
-	if q.Pagination.Page > 0 {
+	if q.Pagination.Cursor.IsEmpty() == false {
 		qs.WriteString("\n")
-		qs.WriteString("LIMIT @offset, @count")
+		qs.WriteString("FILTER ")
```

Comment:
There is a day, FQL query fluent generator (like [squirrel](https://github.com/Masterminds/squirrel)) will be here.

Reply:
Sounds like an idea for a weekend project ;)